lib/webpacker/commands.rb

```diff
@@ -11,6 +11,7 @@ def clobber
   end
 
   def bootstrap
+    return unless config.config_path.exist?
```

Comment:
@dhh @gauravtiwari How about if I submit another PR with the check only here, and print a  warning message?

At this level of the code, is a "puts" to stdout OK?